src/inmanta/deploy/scheduler.py

```diff
@@ -1199,9 +1205,23 @@ async def _update_scheduler_state_for_finished_deploy(
                 state.blocked = BlockedStatus.TRANSIENT
                 # Remove this resource from the dirty set when we block it
                 self._state.dirty.discard(resource)
+            # TODO: consider whether we also want to log on a failed deploy
             elif deployment_result is DeploymentResult.DEPLOYED:
```

Comment:
Question for reviewers: do we also want this warning when a resource *fails* while we expect it to skip for dependencies. I seem to recall we discussed this before and that the conclusion was to only raise it on success, because a failure could be caused by any number of things, and could in fact happen before the dependencies are even checked, which would be valid.

However, @edvgui seemed to wish it to be emitted also for failure, and I promised we'd consider it. I'm personally still in favor of leaving it as is, for the reasons outlined above, and that a use case where the message might be helpful to the user in a failure case all seem like extreme edge cases to me.

Reply:
(failure can also be failure to communicate with the executor, to load the code, ...)

Reply:
I'm in favor of communicating only on success.